java/vector/src/main/java/org/apache/arrow/vector/ZeroVector.java

```diff
@@ -244,4 +244,14 @@ public int getNullCount() {
   public boolean isNull(int index) {
     return false;
   }
+
+  @Override
+  public int hashCode(int index) {
+    return 0;
+  }
+
+  @Override
+  public boolean equals(int index, ValueVector to, int toIndex) {
+    return false;
```

Comment:
this should probably be isNull on to?

Reply:
ZeroVector#getObject will always return null, so equals should always return false?